extensions/ql-vscode/src/view/variant-analysis/RepositoriesResultFormat.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { useCallback } from "react";
+import { styled } from "styled-components";
+import { VSCodeDropdown, VSCodeOption } from "@vscode/webview-ui-toolkit/react";
+import { Codicon } from "../common";
+import { ResultFormat } from "../../variant-analysis/shared/variant-analysis-result-format";
+
+const Dropdown = styled(VSCodeDropdown)`
+  width: 100%;
+`;
+
+type Props = {
+  value: ResultFormat;
+  onChange: (value: ResultFormat) => void;
+
+  className?: string;
+};
+
+export const RepositoriesResultFormat = ({
+  value,
+  onChange,
+  className,
+}: Props) => {
+  const handleInput = useCallback(
+    (e: InputEvent) => {
+      const target = e.target as HTMLSelectElement;
+
+      onChange(target.value as ResultFormat);
+    },
+    [onChange],
+  );
+
+  return (
+    <Dropdown value={value} onInput={handleInput} className={className}>
+      <Codicon name="table" label="Result format..." slot="indicator" />
+      <VSCodeOption value={ResultFormat.Alerts}>Alerts</VSCodeOption>
```

Comment:
I think we can use the enum values here too since they have the same text as we want. 

```suggestion
      <VSCodeOption value={ResultFormat.Alerts}>{ResultFormat.Alerts}</VSCodeOption>
```

BTW, I don't remember using string enum values with spaces in them 🤔 I've looked in our repo and it doesn't look like we're following any kind of convention though (some are capital case, some lower case, some camel case, some pascal case, and in some rare occasions we have spaces 😓 ).

Reply:
Thanks! And yeah, it looks like we're pretty inconsistent with enums 🙈